src/server-node.js

```diff
@@ -507,11 +507,11 @@ function getDnRE(socket) {
       // sliced => *.max.rethinkdns.com
       entry = entry.slice(SAN_DNS_PREFIX.length);
 
-      // d => *\\.max\\.rethinkdns\\.com
+      // d => *\.max\.rethinkdns\.com
       // wc => true
       // pos => 1
-      // match => [a-z0-9-_]*\\.max\\.rethinkdns\\.com
-      const d = entry.replace(/\./g, "\\.");
+      // match => [a-z0-9-_]*\.max\.rethinkdns\.com
+      const d = entry.replaceAll(".", "\\.");
```

Comment:
thanks. can you add comment for lines 527/528. I never quite figured out why those exist (:

Reply:
done.

also a relevant commit: b4938e2